File: 2190-count-common-words-with-one-occurrence/2190-count-common-words-with-one-occurrence.py
import collections

class Solution:
    def countWords(self, words1: List[str], words2: List[str]) -> int:
        w1 = collections.Counter(words1)
        w2 = collections.Counter(words2)
        answer = 0 
        for k in w1.keys():
            if w1[k] == 1 and w2[k] == 1:
                answer += 1
        return answer

File: 2190-count-common-words-with-one-occurrence/2190-count-common-words-with-one-occurrence.go
func countWords(words1 []string, words2 []string) int {
    w1 := make(map[string]int)
    w2 := make(map[string]int)
    answer := 0
    for _, word1 := range(words1) {
        w1[word1] ++
    }
    for _, word2 := range(words2) {
        w2[word2] ++
    }
    for key,_ := range(w1) {
        if w1[key] == 1 && w2[key] == 1 {
            answer += 1
        }
    }
    return answer
}